README.rst

```diff
@@ -314,7 +315,7 @@ Python
 * `cookiecutter-quokka-module`_: A template to create a blueprint module for Quokka Flask CMS.
 * `cookiecutter-kivy`_: A template for NUI applications built upon the kivy python-framework.
 * `cookiedozer`_: A template for Python Kivy apps ready to be deployed to android devices with Buildozer.
-* `cookiecutter-ansible-role`_: A template to create ansible roles. Forget about file creation and focus on actions.
+* `cookiecutter-pypackage-minimal`_: A minimal Python package template.
```

Comment:
This (and https://github.com/audreyr/cookiecutter/pull/903/files#diff-88b99bb28683bd5b7e3a204826ead112R354) looks like a duplicate entry?